home-manager/po/fr.po

```diff
@@ -41,72 +45,78 @@ msgid_plural ""
 "There are %d unread and relevant news items.\n"
 "Read them by running the command \"%s news\"."
 msgstr[0] ""
+"Il y a %d nouvel élément non lu et pertinent.\n"
+"Vous pouvez le lire en exécutant la commande \"%s news\"."
 msgstr[1] ""
+"Il y a %d nouveaux éléments non lus et pertinents.\n"
+"Vous pouvez les lire en exécutant la commande \"%s news\"."
 
 #: home-manager/home-manager:234
 msgid "Unknown \"news.display\" setting \"%s\"."
-msgstr ""
+msgstr "Configuration \"news.display\" iconnue \"%s\"."
```

Comment:
Petite erreur ici 😅

Reply:
I think the contributor in this case is just some kind person that does translations on Weblate so will not see PR comments. If you can then improvement could be done for [this string directly in Weblate](https://hosted.weblate.org/translate/home-manager/cli/fr/?checksum=f9463a0e25881afd). After some time (24h I think) the PR should be updated.